src/views/v2/emails/rsvp.php

```diff
@@ -8,18 +8,26 @@
  * See more documentation about our views templating system.
  *
  * @link https://evnt.is/tickets-emails-tpl Help article for Tickets Emails template files.
+ * If you are looking for Event related templates, see in The Events Calendar plugin.
  *
  * @version 5.5.9
  *
  * @since 5.5.9
  *
- * @var Tribe_Template  $this  Current template object.
+ * @var Tribe_Template $this               Current template object.
```

Comment:
```suggestion
 * @var Tribe__Template $this               Current template object.
```

Let's do a search replace for all the changes as this typo is happening along many templates.